src/SearchInput/SearchInput.tsx

```diff
@@ -58,18 +78,31 @@ export const SearchInput = forwardRef<HTMLInputElement, SearchInputProps>(
       initialState: null,
       stateProp: value,
     })
-
     const [searchQuery, setSearchQuery] = useState<string | null>(null)
 
+    const fuse = useMemo(() => {
+      const searchKeys = ['label', 'value']
+      return new Fuse(searchList, {
+        shouldSort: true,
```

Comment:
Nit: this is set to true by default unless you want to be explicit? [here](https://www.fusejs.io/api/options.html)